mediaplatform_jwp/jwplatformapi/deliveryapi.py

```diff
@@ -16,7 +16,7 @@
 import jwt
 
```

Comment:
suggest calling this module delivery.py given api is in the parent directory

Reply:
agree